app/steps/medicaid/income_other_income_type.rb

```diff
@@ -11,15 +11,14 @@ class IncomeOtherIncomeType < Step
       :member_id,
     )
 
-    def valid?
+    validate :income_type_selected
+
+    def income_type_selected
```

Comment:
would it be more consistent to name this `members_valid` like we have in many of the other steps?